cmd/deletetree/main.go

```diff
@@ -50,8 +51,15 @@ func main() {
 	defer conn.Close()
 
 	a := trillian.NewTrillianAdminClient(conn)
-	_, err = a.DeleteTree(context.Background(), &trillian.DeleteTreeRequest{TreeId: *logID})
-	if err != nil {
-		glog.Exitf("Delete failed: %v", err)
+	if *undeleteTree == false {
```

Comment:
There's a presubmit lint warning here, could you fix it please?

cmd/deletetree/main.go:54:5: S1002: should omit comparison to bool constant, can be simplified to `!*undeleteTree` (gosimple)

Reply:
I've fixed this in my latest commit _many months later_.